### ISSUE TITLE
Support data transformation for outputs/inputs of `call` and `invoke` cheatcodes

### ISSUE THREAD

**Issue by Arcticae** [software-mansion/protostar#1414]
We should consider supporting the transformation on cairo data, so the inputs/outputs are not raw arrays

**Comment by Arcticae** [comment]
I think this might be worth holding off, until we know what's the `call` interface for cairo 1.0. Moving this to improvements milestone.

**Comment by Arcticae** [comment]
Blocked by [cairo 1.0 integration](https://github.com/software-mansion/protostar/milestone/31)

**Comment by MaksymilianDemitraszek** [comment]
We probably want to wait until StarkWare implements call syscalls so we can implement similar interface for cheatcodes for simplicity. Internal interface will be impacted by external as it is hard to predict requirements atm

**Comment by Arcticae** [comment]
Yep but still we need cairo 1.0 for this, so this is just a matter of integration status + when SW implements proper syscall handling logic (then we'd just update the compiler, and this will be ready to do). 

Although I suppose cairo 1.0 integration can't be considered "finished" until it has feature parity with old cairo, so "blocked by cairo 1.0" seems like equivalent of the above.
